src/app/api/services/dot-router/dot-router.service.spec.ts

```diff
@@ -57,6 +57,12 @@ describe('DotRouterService', () => {
         expect(service.goToEditPage).toHaveBeenCalledWith('/about/us');
```

Comment:
![Screen Shot 2019-09-05 at 7 59 46 AM](https://user-images.githubusercontent.com/751424/64348925-4548b800-cfb3-11e9-9ff2-e5b2489d1ea7.png)

Since is just one options we should not show the sub menu just do the action when user clicks the button.

Here you can pass a command instead of a model: https://github.com/dotCMS/core-web/blob/issue-17099-create-form-module-and-routing/src/app/shared/models/action-header/action-header-options.model.ts#L4